### PR TITLE
adding databroker-pack & suitcase recipes

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/databroker-pack-feedstock
https://github.com/nsls-ii-forge/suitcase-csv-feedstock
https://github.com/nsls-ii-forge/suitcase-json-metadata-feedstock
https://github.com/nsls-ii-forge/suitcase-specfile-feedstock
https://github.com/nsls-ii-forge/suitcase-jsonl-feedstock
https://github.com/nsls-ii-forge/suitcase-tiff-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

File: recipes/suitcase-csv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (mrakitin) [inline recipes/suitcase-csv/meta.yaml:16]: Let's synchronize the commands used across the recipes. I'd suggest using the one from the databroker-pack's recipe, that also matches the recommendation from https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip (I excluded double-quotes as the auto-tick bot does that anyway):

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/suitcase-json-metadata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (mrakitin) [inline recipes/suitcase-json-metadata/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/suitcase-jsonl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
```
Comment (mrakitin) [inline recipes/suitcase-jsonl/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/databroker-pack/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - databroker-pack = databroker_pack.commandline.pack:main
+    - databroker-unpack = databroker_pack.commandline.unpack:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (mrakitin) [inline recipes/databroker-pack/meta.yaml:19]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/suitcase-specfile/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (mrakitin) [inline recipes/suitcase-specfile/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/suitcase-tiff/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
```
Comment (mrakitin) [inline recipes/suitcase-tiff/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

**Review by mrakitin — COMMENTED** [review]
Thanks, @kbeeperez! I left a few suggestions.

To the reviewers, I agree to be a maintainer of all feedstocks for the recipes listed here.

**Review by mrakitin — APPROVED** [review]

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes could this be merged soon? They are dependencies for other packages we need.

File: recipes/databroker-pack/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
+{% set name = "databroker-pack" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "2a69d38bc23ed3b6fc7d017fead5a6096a99428858dd8b77568d78309f9d0a1b" %}
```
Comment (ocefpaf) [inline recipes/databroker-pack/meta.yaml:3]: Same comment as the other PRs ;-p

File: recipes/suitcase-csv/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,45 @@
+{% set name = "suitcase-csv" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "2f8eece9502084dcd81143e8ff95ad15ead43325f2fc063f0642789ffecd8a8f" %}
```
Comment (mrakitin) [inline recipes/suitcase-csv/meta.yaml:3]: @kbeeperez, following @ocefpaf's suggestion above, do you mind updating this place and the rest of the recipes in this PR to have the sha256 in the `source` block? Thanks!

**Comment by kbeeperez** [comment]
@ocefpaf should be good to merge now.

**Pull request merged**